sponsors.html

```diff
@@ -4,7 +4,19 @@
 ---
 <div class="container white-text">
   <h1 class="text-accent-4 center">Sponsor</h1>
-
+  <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black update clemson-orange-text">Online update for Sponsors</div>
+      <div class="collapsible-body">
+        <p>With CUCTF 2020 switching to an online infrastructure all Silver, Gold, and Diamond sponsors will:</p>
+        <ul class="browser-default">
+          <li>Be provided private online meeting lobbies to connect with students and present information about their organization</li>
+          <li>Maintain sponsorship status into the CUCTF 2021 Event</li>
+        </ul>
+        <p>CU Cyber understands that interacting with students online does not match the value of meeting with students in person. To ensure that your donation gives you your full value, Silver, Gold, and Diamond sponsors will have the option to maintain their sponsor status at CUCTF 2021 with no extra cost!</p>
```

Comment:
No extra cost, as in they have to give next year but no extra amount, or just no cost?

Reply:
I rephrased the sentence, but I trust the sponsor's intelligence to understand the context of the sentence.